### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by frankiebee**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Project Name *optional*:
+
+Github Repo:<!--
+not necessary to fill this out now you can come back and
+fill this out when it's time to turn in your work
+or comment on this issue with the link.
+-->
+
+
+<!-- Delete this section if you do not have team mates -->
```
Comment (2-am-zzz) [inline ISSUE_TEMPLATE.md]: teammates

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+
+
+###Your Info:
+<!-- The one you want us to call you when you win -->
```
Comment (2-am-zzz) [inline ISSUE_TEMPLATE.md:5]: I dunno if the comments are necessary, but it's nice for clarity.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+not necessary to fill this out now you can come back and
+fill this out when it's time to turn in your work
+or comment on this issue with the link.
+-->
+
+
+<!-- Delete this section if you do not have team mates -->
+###Team Info:
+
+Team mates :
```
Comment (2-am-zzz) [inline ISSUE_TEMPLATE.md]: teammates

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<!--
+Please provide the same info for your team mates that
+you provide for your self. Excluding the project name of course. As well as include
```
Comment (2-am-zzz) [inline ISSUE_TEMPLATE.md]: yourself

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
-# IPFS-Ethereum-Hackathon
-Repo for hackathon submissions 
+# IPFS Ethereum Hackathon 2017!
+### February 5th to 11th
+
+
+Welcome to the first ever IPFS Ethereum Hackathon! Brought to you by your friendly
+neighborhood MetaMask.
+
+## Schedule and Over view:
```
Comment (2-am-zzz) [inline README.md]: overview

File: README.md [hunk 6]
```diff
@@ -1,2 +1,55 @@
...
+This is a remote Hackathon. You do not need to be in person for the over all hackathon.
```
Comment (2-am-zzz) [inline README.md]: overall

File: README.md [hunk 7]
```diff
@@ -1,2 +1,55 @@
...
+However their is an in person options for some of the events happening in the week (location Seattle)
+More information on this can be found [here](https://www.meetup.com/Seattle-IPFS-Meetup/events/236101964/)
+##### Quick over view of schedule
+| Date  | Event              | Start Time(PST) |
+| ----- | ------------------ | --------------- |
+| 02/5  | [Kick off and Talks](https://www.meetup.com/Seattle-Ethereum-Meetup/events/237147519/) | 6:30pm    |
+| 02/8  | [Cafe drop in/Open office Ours]() | 9:30am     |
```
Comment (2-am-zzz) [inline README.md]: Open office hours

File: README.md [hunk 8]
```diff
@@ -1,2 +1,55 @@
...
+| 02/10 | [Dead line]() | 1:00pm    |
```
Comment (2-am-zzz) [inline README.md]: deadline

File: README.md [hunk 9]
```diff
@@ -1,2 +1,55 @@
...
+## Channels of communication
+If you need help or have questions you can find us here:
+- MetaMask Slack in [#ipfs-ethereum-hack](https://metamask.slack.com/messages/ipfs-ethereum-hack/) not a member? sign up [here](https://metamask-slack-autoinvite.herokuapp.com/)
+- [gitter](https://gitter.im/MetaMask/Lobby)
+
+## Rules and Code of conduct:
+- Don't be mean
+
+## Thank yous!
+We would to and recognize some groups and people who made this happen.
```
Comment (danfinlay) [inline README.md]: We would like to recognize some groups and people who made this happen.

File: SCORING_GUIDE.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+- How unique is the idea?
+
+#####The MVP:
+- Does it make sense when you use it?
+- Does it look nice?
+- Does it impress you?
+- Would it make sense to the average user?
+
+#####Implementation:
+- Did they use the Ethereum block chain and IPFS?
```
Comment (2-am-zzz) [inline SCORING_GUIDE.md]: blockchain

File: SCORING_GUIDE.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+# Scoring Guide Lines
+
+### Score Table:
+
+|            | Idea | MVP | Implementation of the Tech | Total |
+| ---------- | ---- | --- | -------------------------- | ----- |
+| Max Score  |   20 |  50 |                         30 |   100 |
+
+
+###Things too look for while scoring:
```
Comment (2-am-zzz) [inline SCORING_GUIDE.md]: to look for

File: README.md [hunk 12]
```diff
@@ -1,2 +1,55 @@
...
+
+
+## How to register:
+Open an Issue on this repo with this info:
+```
+  Your name:
+  Your project/team name:
+
+  A short description of the project:
+  Repo link: (optional When registering but necessary before dead line )
```
Comment (2-am-zzz) [inline README.md]: deadline

File: README.md (hunk 6 above)
Comment (danfinlay) [inline README.md]: overall: one word

**Review by 2-am-zzz — CHANGES_REQUESTED** [review]

File: README.md [hunk 13]
```diff
@@ -1,2 +1,55 @@
...
+# IPFS Ethereum Hackathon 2017!
+### February 5th to 11th
+
+
+Welcome to the first ever IPFS Ethereum Hackathon! Brought to you by your friendly
+neighborhood MetaMask.
+
+## Schedule and Over view:
+This is a remote Hackathon. You do not need to be in person for the over all hackathon.
+However their is an in person options for some of the events happening in the week (location Seattle)
```
Comment (danfinlay) [inline README.md]: However there are in person options for some of the events happening in the week (location Seattle)

**Review by danfinlay — CHANGES_REQUESTED** [review]
Just a few lines with typos that I noticed.  Otherwise, really awesome hackathon repo!

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (frankiebee) [reply ISSUE_TEMPLATE.md:5]: Keeping the comments i think it makes things easy

**Pull request merged**